templates/includes/legalcode_licenses_4.0.html

```diff
@@ -599,11 +581,7 @@ <h3 id="s3">
 
 {% if legal_code|is_one_of:"by-nc-sa,by-sa" %}
 <li id="s3b">
-<strong>
-{% blocktrans trimmed %}
-ShareAlike
-{% endblocktrans %}
-</strong>.
+<strong>{% blocktrans trimmed %}ShareAlike{% endblocktrans %}</strong>.
```

Comment:
@TimidRobot, should this (and "Attribution" and "Other Rights" and other similar phrases) also be using `trans` instead of `blocktrans` now that each one is on a single line?  I expect things would be a little easier to read that way.